config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Annotation.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.beans;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.Collection;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Annotation {
+    private String type;
+    private String name;
+    private String description;
+    private Collection<String> tags;
+
+    public Annotation(final String type) {
+        this.type = type;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public Collection<String> getTags() {
+        return tags;
+    }
+
+    public void setTags(Collection<String> tags) {
+        this.tags = tags;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Annotation that = (Annotation) o;
+
+        return type.equals(that.type);
```

Comment:
do we need not consider name and other properties

Reply:
I don't think so, only `@bundle` is supposed to have `name` and we are not supporting multiple annotations of same type. For example, we cannot have 2 `@bundle` annotations.